projects/kit/components/breadcrumbs/breadcrumbs.template.html

```diff
@@ -1,26 +1,9 @@
-<ng-container *ngFor="let item of items; last as lastItem">
-    <ng-container *tuiLet="item | tuiMapper: routerLinkOptions as options">
```

Comment:
Should we also delete imports of `TuiMapperPipeModule` / `TuiLetModule` / `TuiLinkModule` / `RouterModule`  from the module of the component?

I am not sure that we need to delete all of them. May be some ones are redudant.

Reply:
yes, thanks!